### DISCUSSION TITLE
How to solve [FindTransferAnchors] invalid or not-yet-implemented 'Matrix' subsetting problem

### DISCUSSION THREAD

**Discussion by kimjh0107** [satijalab/seurat#4859] [General]
Hi, thank you for making a wonderful tools. 
I have a question about the function  " FindTransferAnchors( ) " used in the multimodal reference mapping tutorial. 

I am trying to get map one of my data to the reference data I am using. I used SCTransform tutorial method to my reference data and now I am trying to map my query data, but I've got in a problem. Whenever I used FindTransferAnchors( ) function to make a anchors the error : 'Error in umi[residual.features, , drop = FALSE] : invalid or not-yet-implemented 'Matrix' subsetting' comes up. Is there any way to solve this problem? 

What I found is that both of my reference and query data matrix format is in 'dgCMatrix'. When I looked at the tutorial data, the data matrix format was 'dfCMatrix'. Is this the reason that I cannot get into the next step right now? 

+ Right now I am using the latest version of Seurat. 

![image](https://user-images.githubusercontent.com/83206535/127446521-8c5f4ba9-a257-4083-a161-752f604b9b6c.png)

**Comment by timoast** [comment]
Please open an issue and include the full code you're running, as well as the output of `sessionInfo()`
